root/static/scripts/common/components/FingerprintTable.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React, {useEffect, useState} from 'react';
+
+import loopParity from '../../../../utility/loopParity';
+import hydrate from '../../../../utility/hydrate';
+
+function orderTracks(tracks) {
+  tracks.sort(function (a, b) {
```

Comment:
In my previous comment I meant just moving this inner function that accepts `(a, b)` outside, but keeping the `tracks.sort` call in the component, so that it doesn't need to create a closure at all.